README.md

```diff
@@ -30,10 +25,36 @@ PK-Sim® uses building blocks that are grouped into [**Individuals**](https://do
 
-### Toolboxes for Matlab and R
-The MoBi® toolboxes for [R](https://docs.open-systems-pharmacology.org/working-with-matlab-and-r/r-introduction) and [Matlab®](https://docs.open-systems-pharmacology.org/working-with-matlab-and-r/matlab-introduction) are interfaces to the common statistical and technical computing environments, respectively. Basically, the toolboxes can be used to access and modify model parameters as well as to execute simulations and retrieve results. That way, the toolboxes can be used to script or code batch simulations, analysis tasks, or customized workflows to any complexity. Results can be visualized using the options available in the respective environment. Apart from the  communication and exchange via Matlab®, PK-Sim® and MoBi® have import and export functions that allow for the import of experimental data via MS Excel® and models based on SBML or the export of simulation results via MS Excel®, for example. 
+- a different set of starting values may define a new scenario, situation, or individual. 
+- refine a Reaction(s) network and update it in all tissues where it should be considered.
+
+### Qualification framework
+
+The qualification framework enables an automated validation of various scenarios (use-cases) supported by the OSP platform. This technical framework is used, for example, to release, in full confidence, a new version of the OSP Suite by verifying automatically that an ever-growing list of scenarios is performing as expected. Qualification framework is described in detail [here](https://docs.open-systems-pharmacology.org/shared-tools-and-example-workflows/qualification).
+
+### Validation and automation tools
+
+Validation and automation tools include for example:
+
+* **Installation Validator**: enables "1-Click" validation of the OSP Suite installation on a target computer. The validation is performed by execution of the predefined set of simulation scenarios and comparison of the simulated results with the (validated) reference values.
+* **Command Line Interface** (*CLI*): allows batch processing of multiple projects in PK-Sim and is described in [Command Line Interface - CLI](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-command-line-interface).
+
+### Toolboxes
+
+The MoBi® toolbox for R is an interface to the common statistical computing environment. Basically, the R-toolbox can be used to access and modify model parameters as well as to execute simulations and retrieve results. That way, the R-toolbox can be used to script or code batch simulations, analysis tasks, or customized workflows to any complexity. Results can be visualized using the options available in the respective environment. The MoBi® toolbox for R is described in detail in [R documentation](https://docs.open-systems-pharmacology.org/working-with-r/r-introduction).
```

Comment:
Let's not call it the mobi tool box. It has nothing to do with Mobi anymore

Reply:
OSP Toolbox for R?

Reply:
changed to OSP toolbox for R